pkg/ctl/create/cluster.go

```diff
@@ -94,9 +97,58 @@ func createClusterCmd() *cobra.Command {
 
 	fs.BoolVarP(&ng.PrivateNetworking, "node-private-networking", "P", false, "whether to make initial nodegroup networking private")
 
+	groupFlagsInUsage(cmd)
+
 	return cmd
 }
 
+func groupFlagsInUsage(cmd *cobra.Command) {
+	// Group flags by their categories determined by name prefixes
+	groupToPatterns := map[string][]string{
```

Comment:
Eventually would be nice to automate this, so we don't have to keep two copies, but I'm happy to merge this as is to start with!